source/Calamari.Tests/TargetDiscoveryBehaviourIntegrationTestFixture.cs

```diff
@@ -71,7 +73,7 @@ public async Task Setup()
             webMgmtClient = new WebSiteManagementClient(new TokenCredentials(authToken))
             {
                 SubscriptionId = subscriptionId,
-                HttpClient = { BaseAddress = new Uri(DefaultVariables.ResourceManagementEndpoint) },
+                HttpClient = { BaseAddress = new Uri(DefaultVariables.ResourceManagementEndpoint), Timeout = TimeSpan.FromMinutes(5) },
```

Comment:
This timeout was added in response to a bunch of test failures where calls to update web apps/slots were timing out after 100 seconds (default timeout), it's a bit yuck, not sure if there are other options we could look at, just seemed like Azure was having problems with what we were throwing at it unfortunately.